src/bindings/python/src/openvino/__init__.py

```diff
@@ -12,47 +12,47 @@
 except ImportError:
     pass
 
-# API 2.0
-try:
-    # Import all public modules
-    from openvino import runtime as runtime
-    from openvino import frontend as frontend
-    from openvino import helpers as helpers
-    from openvino import preprocess as preprocess
-    from openvino import utils as utils
-    from openvino.runtime import properties as properties
-
-    # Import most important classes and functions from openvino.runtime
-    from openvino.runtime import Model
-    from openvino.runtime import Core
-    from openvino.runtime import CompiledModel
-    from openvino.runtime import InferRequest
-    from openvino.runtime import AsyncInferQueue
-
-    from openvino.runtime import Dimension
-    from openvino.runtime import Strides
-    from openvino.runtime import PartialShape
-    from openvino.runtime import Shape
-    from openvino.runtime import Layout
-    from openvino.runtime import Type
-    from openvino.runtime import Tensor
-    from openvino.runtime import OVAny
-
-    from openvino.runtime import compile_model
-    from openvino.runtime import get_batch
-    from openvino.runtime import set_batch
-    from openvino.runtime import serialize
-    from openvino.runtime import shutdown
-    from openvino.runtime import tensor_from_file
-    from openvino.runtime import save_model
-    from openvino.runtime import layout_helpers
-
-    # Set version for openvino package
-    from openvino.runtime import get_version
-    __version__ = get_version()
-except ImportError:
-    import warnings
-    warnings.warn("openvino package has problems with imports!", ImportWarning, stacklevel=2)
+# #
+# # API 2.0
+# # This __init__.py forces checking of runtime modules to propagate errors.
+# # It is not compared with init files from openvino-dev package.
+# #
+# Import all public modules
+from openvino import runtime as runtime
+from openvino import frontend as frontend
+from openvino import helpers as helpers
+from openvino import preprocess as preprocess
+from openvino import utils as utils
+from openvino.runtime import properties as properties
+
+# Import most important classes and functions from openvino.runtime
+from openvino.runtime import Model
+from openvino.runtime import Core
+from openvino.runtime import CompiledModel
+from openvino.runtime import InferRequest
+from openvino.runtime import AsyncInferQueue
+
+from openvino.runtime import Dimension
+from openvino.runtime import Strides
+from openvino.runtime import PartialShape
+from openvino.runtime import Shape
+from openvino.runtime import Layout
+from openvino.runtime import Type
+from openvino.runtime import Tensor
+from openvino.runtime import OVAny
+
+from openvino.runtime import compile_model
+from openvino.runtime import get_batch
+from openvino.runtime import set_batch
+from openvino.runtime import serialize
+from openvino.runtime import shutdown
+from openvino.runtime import tensor_from_file
+from openvino.runtime import save_model
```

Comment:
Should we have `convert_model` here?

Reply:
it's below in a separate try / catch block.

This is because if I use only python API from build folder, OVC is not there